### PR TITLE
14604 court order file

### PR DESCRIPTION
**Pull request by vysakh-menon-aot**
*Issue #:* /bcgov/entity#14604

*Description of changes:*
  - fileName is not required in the JSON anymore as we don't do anything with it (based on the discussion with @thorwolpert) 
  - Clean up of example data (COOP_INCORPORATION)

By submitting this pull request, I confirm that you can use, modify, copy, and redistribute this contribution, under the terms of the business-schemas license (Apache 2.0).

### CODE REVIEW

**Review by ChrisGabel23 — APPROVED** [review]
I would prefer to see a description of the changes made in this PR.

File: tests/unit/test_court_order.py [hunk 1]
```diff
@@ -38,6 +38,8 @@ def test_validate_valid_court_order():
     del order_json['effectOfOrder']
     del order_json['orderDate']
     del order_json['orderDetails']
+    del order_json['fileKey']
+    del order_json['fileName']
```
Comment (severinbeauvais) [inline tests/unit/test_court_order.py:42]: See also the UI code to for other properties that should be stored.

Ref: https://github.com/bcgov/business-filings-ui/blob/c7fadad6ca076a2e6497af991891c8e7b9def7ec/src/components/dialogs/AddStaffNotationDialog.vue#L328

**Comment by vysakh-menon-aot** [comment]
> I would prefer to see a description of the changes made in this PR.

It should be part of the issue referred in each pr. If we do anything addition to that we include it in description.

File: tests/unit/test_court_order.py (hunk 1 above)
Comment (vysakh-menon-aot) [reply tests/unit/test_court_order.py]: fileLastModified and fileSize are not expected by API. Is there any new requirement? and the fileName is not required any more as we don't do anything with it (based on the discussion with @thorwolpert)

Comment (severinbeauvais) [reply tests/unit/test_court_order.py]: Oh ok. I guess they're still in the filing JSON anyway.

**Comment by severinbeauvais** [comment]
> It should be part of the issue referred in each pr. If we do anything addition to that we include it in description.

Could you please check off the completed todo items in the ticket, and move the ticket to the Review column? Thanks!

Does/should the filing schema need to change in any way?

File: src/registry_schemas/schemas/court_order.json [hunk 2]
```diff
@@ -37,6 +37,10 @@
                     "minLength": 0,
                     "maxLength": 2000,
                     "description": "A brief note to explain the purpose of the Court Order."
+                },
+                "fileKey": {
+                    "type": "string",
+                    "title": "The Identifier for court order file in file server"
```
Comment (severinbeauvais) [inline src/registry_schemas/schemas/court_order.json:43]: Should this, or any of the other properties, be "required" ?

**Review by severinbeauvais — APPROVED** [review]

File: src/registry_schemas/schemas/court_order.json (hunk 2 above)
Comment (vysakh-menon-aot) [reply src/registry_schemas/schemas/court_order.json:43]: Custom validation is in legal-api

Comment (severinbeauvais) [reply src/registry_schemas/schemas/court_order.json:43]: Yes but required fields can be validated by this schema. Are any fields required for a court order?

Comment (vysakh-menon-aot) [reply src/registry_schemas/schemas/court_order.json:43]: Required fields are different according to the type of filing. For Court order filing (staff), orderDetails is required and its not required in alteration or dissolution.

Comment (severinbeauvais) [reply src/registry_schemas/schemas/court_order.json:43]: Oh, gotcha.

OK, lgtm.

**Review by thorwolpert — APPROVED** [review]

**Pull request merged**